src/index.js

```diff
@@ -1,24 +1,20 @@
+import addScore from './modules/addScores.js';
+import getScores from './modules/getScores.js';
 import './style.css';
-import addLi from './modules/add-li.js';
 
-const leaderboardData = {
-  result: [
-    {
-      user: 'John Doe',
-      score: 42,
-    },
-    {
-      user: 'Peter Parker',
-      score: 35,
-    },
-    {
-      user: 'Wonder Woman',
-      score: 50,
-    },
-  ],
-};
+const scoresList = document.querySelector('.scores-list');
+const submitButton = document.querySelector('.submit-button');
+const refreshButton = document.querySelector('.refresh');
+const message = document.querySelector('.message');
 
-leaderboardData.result.forEach((element) => {
-  const text = `${element.user}: ${element.score}`;
-  addLi(text);
+refreshButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  getScores(scoresList);
+});
+
+submitButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  const user = document.querySelector('.user').value;
+  const score = document.querySelector('.score').value;
+  addScore(scoresList, user, score, message);
```

Comment:
Currently, your `addEventListener` isnt working, as seen on the error:
![image](https://user-images.githubusercontent.com/42799579/197360137-bf5ef5e5-d6aa-4997-b0bd-70b33d2785d5.png)

This is happening because you are referencing the wrong elements on your querySelector. Please adjust the name of the ids, as in the screenshot:
![image](https://user-images.githubusercontent.com/42799579/197360195-7ee8c107-07c0-468c-8d0e-90ceecabb188.png)

## IMPORTANT NOTE:
I've noticed that your project is working as intended, as seen in the next screenshot:
![image](https://user-images.githubusercontent.com/42799579/197360449-5974adac-825b-43b1-b234-1e9c23051daa.png)

Although, all your references are linked to classes/ids that don't exist on your code. Please adjust the name of all the references to achieve the desired result. 😄